Documentation/HowToGuides/PassingAFloatValueFromAnInteractorToAnInteractable/README.md

```diff
@@ -0,0 +1,190 @@
+
+![Drag And Drop Action Publisher Into Left Controller Alias](assets/images/DragAndDropActionPublisherIntoLeftControllerAlias.png)
+
+Expand the `Input.UnityInputManager.Oculus.TouchLeftController -> Input Actions -> LeftTrigger` until you can see the `LeftTrigger_Axis[9]` GameObject, select the `Interactions.ActionPublisher` and drag and drop the `LeftTrigger_Axis[9]` GameObject into the `Source Action` property on the `Interactor Action Publisher Facade` component.
+
+![Drag And Drop Left Axis9 Into Source Property](assets/images/DragAndDropLeftAxis9IntoSourceProperty.png)
+
+> Do the same with the right controller.
+
+### Step 5
+
+Drag and drop the `Interactions.Interactor` GameObject embedded within the `LeftControllerAlias` into the `Source Interactor` property on the `Interactor Action Publisher Facade` component.
+
+![Drag And Drop Interactions Interactor Into Source Publisher](assets/images/DragAndDropInteractionsInteractorIntoSourcePublisher.png)
+
+> Do the same with the right controller.
+
+### Step 6
+
+The `Publisher Indentifier` property is what groups the action publishers to the same final action receiver, so for these two triggers we want them to have the same publisher identifier value. Set the `Publisher Identifier` for both the left `Interactions.ActionPublisher` and the right `Interactions.ActionPublisher` to `TriggerPublisher`.
+
+![Rename Publisher Trigger](assets/images/RenamePublisherTrigger.png)
+
+### Step 7
+
+Before we set up the Interactable lets create a container GameObject for all this logic. Create a new GameObject and name it `OpacityChangingInteractable`. Set its position to match that of the `Interactions.Interactable` GameObject.
+
+Position: `X = 0`, `Y = 0.6`, `Z = 0`
+
+![Create Opacity Changing Interactable](assets/images/CreateOpacityChangingInteractable.png)
+
+then drag and drop the `Interactions.Interactable` GameObject to be a child of the `empty` GameObject.
+
+![Make Interactions Interactable A Child Of Opacity Changing Interactable](assets/images/MakeInteractionsInteractableAChildOfopacityChangingInteractable.png)
+
+### Step 8
+
+Expand the `Tilia Interactions Interactables Unity` Package directory in the Unity project window until the `Packages -> Tilia Interactions Interactables Unity -> Runtime -> Interactors -> Prefabs`, then drag and drop the `Interactions.ActionReceiver` as a child of the `OpacityChangingInteractable` GameObject
+
+![Drag And Drop Action Receiver](assets/images/DragAndDropActionReceiver.png)
+
+### Step 9
+
+Drag and drop the `Interactions.Interactable` GameObject nested within the `OpacityChangingInteractable` GameObject into the `Target Interactable` property on the `Interactable.ActionReceiver Facade` component on the `OpacityChangingInteractable -> Interactions.ActionReceiver` GameObject.
+
+![Drag And Drop Interactions Interactable Into Target Interactable](assets/images/DragAndDropInteractionsInteractableIntoTargetInteractable.png)
```

Comment:
The arrow is coming from the wrong object, it should be coming from `Interactions.Interactable` and not `interactions.ActionReceiver`

![image](https://user-images.githubusercontent.com/1029673/74438468-23a50400-4e62-11ea-80f1-980922fac1f8.png)